### ISSUE TITLE
Upgrade cmake-rs for AST explorer

### ISSUE THREAD

**Issue by stevefan1999-personal** [immunant/c2rust#402]
The bug is fixed in upstream but is yet released. This is needed to try to build it on Windows, this is so far the obstacle

**Comment by stevefan1999-personal** [comment]
...although using MinGW should work, why not? (don't close yet, it's bout 4am in the morning in local time, lemme test it when I woke)

**Comment by kkysen** [comment]
@stevefan1999-personal, does it work with MinGW?  Regardless, we'll upgrade `cmake-rs` as soon as the next release comes out.

**Comment by stevefan1999-personal** [comment]
@kkysen I hacked it a little to add VS2022. However I met other obstacles in using it from Windows, most notably the lack of POSIX support in Windows makes it rather limited. I was hoping to translate some old C shellcode and the renowned QuakeWin engine using c2rust but it seems like it is not working that well. Even with the simple [OTCC example](https://bellard.org/otcc/), some functions are mysteriously missing.

**Comment by kkysen** [comment]
That's unfortunate.

I also saw someone else, @Shadlock0133, had some trouble with building `c2rust` on Windows due to Windows' LLVM releases not containing `llvm-config` unless you build from source (https://github.com/immunant/c2rust/issues/463).  Did you have any other issues like that?

**Comment by stevefan1999-personal** [comment]
@kkysen Yes indeed. For some reason I failed to contain libgcc and after copying it that works.
